README.md

```diff
@@ -153,6 +153,7 @@ If any error is encounterd, **error** will be set
   * **reply** (number postId, string body, [*binaryString attachment*], function callback)
   * **delete** (number postId, function callback)
   * **rate** (number postId, number score, function callback)
+  * **liked** (string type, number id, [*object options*], function callback)
```

Comment:
change the name of the parameters to be the same as other calls that use this ([*string section*], [*number identifier*]) as with abuse.report